src/read/wasm.rs

```diff
@@ -168,6 +170,23 @@ impl<'data, R: ReadRef<'data>> WasmFile<'data, R> {
                 }
                 wp::Payload::GlobalSection(section) => {
                     file.add_section(SectionId::Global, section.range(), "");
+                    for global in section {
+                        let global = global.read_error("Couldn't read a global item")?;
+                        let mut address = None;
+                        const I32_CONST: wp::GlobalType = wp::GlobalType {
+                            content_type: wp::ValType::I32,
```

Comment:
Should be I64 if the memory is 64bit.

Reply:
Done.

Reply:
This doesn't work completely yet. I have a wasm64 file that has an import instead of having a memory section:
`  (import "env" "memory" (memory (;0;) i64 2))`

How about always allowing both I32 and I64? We're not trying to be a validator, so it doesn't matter if we allow too much.

Reply:
I've simplified the code.